clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.cli.commands;
+
+import org.apache.gravitino.client.GravitinoAdminClient;
+import org.apache.gravitino.client.GravitinoClient;
+import org.apache.gravitino.exceptions.NoSuchMetalakeException;
+
+/* The base for all commands. */
+public abstract class Command {
+  private final String url;
+
+  /**
+   * Command constructor.
+   *
+   * @param url The URL of the Gravitino server.
+   */
+  protected Command(String url) {
+    this.url = url;
+  }
+  /** All commands have a handle method to handle and run the required command. */
+  public abstract void handle();
+
+  /**
+   * Builds a {@link GravitinoClient} instance with the provided server URL and metalake.
+   *
+   * @param metalake The name of the metalake.
+   * @return A configured {@link GravitinoClient} instance.
+   * @throws NoSuchMetalakeException if the specified metalake does not exist.
+   */
+  protected GravitinoClient buildClient(String metalake) throws NoSuchMetalakeException {
+    return GravitinoClient.builder(url).withMetalake(metalake).withVersionCheckDisabled().build();
```

Comment:
It was added so it works with the playground, removing that check means it fails to work with the playground as you get this error:
The client version is 0.7.0-incubating-SNAPSHOT, and the server version 0.6.0-incubating

While that is correct and may cause may cause some issues, it makes it difficult for a user to use the CLI with the playground. Even if the playground was updated to the latest release the version check would still fail.

Reply:
But in practice, we don't recommend users to disable version check, because we currently only guarantee the backward compatibility, the "disable" will make some operation fail, I would suggest you can add a option to control whether we want to disable this, by default it should be enabled.